CMakeLists.txt

```diff
@@ -29,41 +29,35 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-project(otf2xx)
-
 cmake_minimum_required(VERSION 3.5)
 
-set(otf2xx_VERSION 1.0)
+project(otf2xx VERSION 1.0.0)
```

Comment:
> The VERSION option may not be used unless policy CMP0048 is set to NEW.

Reply:
That is the case for CMake >= 3.0:

- https://devdocs.io/cmake~3.5/policy/cmp0048#policy:CMP0048 (Introduced in 3.0)
- https://devdocs.io/cmake~3.5/command/cmake_policy (All policies introduced in the specified version or earlier will be set to use NEW behavior)
- https://devdocs.io/cmake~3.5/command/cmake_minimum_required (command implicitly invokes cmake_policy)

Reply:
I didn't know that cmake_minim_required also sets policies to NEW. I tried it in the past and got the unpleasant error:
```
CMake Error at CMakeLists.txt:2 (project):
  VERSION not allowed unless CMP0048 is set to NEW
```

I probably had `project` and `cmake_minimum_required` in the wrong order :(

Reply:
Yes. They state that in the docu. In this project it was the wrong way round (also fixed here)